### PR TITLE
normalize folders restored in testing

### PR DESCRIPTION
**Pull request by ryanfkeepers**
Normalize all test folders to use a constant prefix: Corso_Test.  This cleans up and centralizes all per- test variations on the restore destination.

---

#### Does this PR need a docs update or release note?

- [x] :no_entry: No

#### Type of change

- [x] :robot: Supportability/Tests

#### Test Plan

- [x] :zap: Unit test
- [x] :green_heart: E2E

### CODE REVIEW

File: src/internal/tester/restore_destination.go [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
 )
 
-func DefaultTestRestoreDestination() control.RestoreDestination {
-	// Use microsecond granularity to help reduce collisions.
-	return control.DefaultRestoreDestination(dttm.SafeForTesting)
+const RestoreFolderPrefix = "Corso_Test"
+
+func DefaultTestRestoreDestination(namespace string) control.RestoreDestination {
+	var (
+		dest = control.DefaultRestoreDestination(dttm.SafeForTesting)
```
Comment (pandeyabs) [inline src/internal/tester/restore_destination.go:14]: Looks like we are overwriting dest on line 23 below. Should we remove this call?

File: src/internal/connector/sharepoint/restore.go [hunk 2]
```diff
@@ -125,7 +125,7 @@ func RestoreCollections(
 }
 
 // restoreListItem utility function restores a List to the siteID.
-// The name is changed to to Corso_Restore_{timeStame}_name
+// The name is changed to to {DestName}_{name}
```
Comment (pandeyabs) [inline src/internal/connector/sharepoint/restore.go:128]: This is prod code. Just to confirm, is this change intended?

**Review by pandeyabs — APPROVED** [review]
Looks good. Thanks for this!

File: src/internal/tester/restore_destination.go (hunk 1 above)
Comment (ryanfkeepers) [reply src/internal/tester/restore_destination.go:14]: Nope, this will populate values beyond just the container name.
